src/chrome/content/rules/Afterdawn.com.xml

```diff
@@ -0,0 +1,21 @@
+<!--
+	Nonfunctional hosts in *.afterdawn.com:
+
+		- afterdawn.com (h)
```

Comment:
This can be added as target. The server redirects to `www` which is secured below.